File: Java/Vowel.java
public class Vowel{
    public static void main(String [] args)
    {
        String str="ptn";
        int count=0;

        for(char c:str.toCharArray())
        {
            if(c=='a' || c=='i' || c=='e' || c=='o' || c=='u')
            {
                count++;
            }
        }
            if(count>0)
            {
                System.out.println("Vowels Present");
            }
            else
            {
                System.out.println("Vowels Not Present");
            }
        
    }
}

File: NodeJS/Vowel.js
function isVowelPresent(str)
{
    const vowels=['a','e','i','o','u','A','E','I','O','U'];

    for(const char of str)
    {
        if(vowels.includes(char))
        {
            return true;
        }
    }
    return false;
}   

const inputstring="stack"

if(isVowelPresent(inputstring))
{
    console.log("Contains Vowel");
}
else
{
    console.log("No Vowels");
}